packages/geolocation/src/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const welcome = require("./utils/welcomePackage")
+
+module.exports = {
+    geolocationWelcome: welcome,
+}
```

Comment:
@Edilberto-Vazquez Can we add the EOL here?

Reply:
I already add the EOL